charts/unifi/0.0.1/questions.yaml

```diff
@@ -362,4 +362,76 @@ questions:
                       schema:
                         type: string
                         default: ""
-                        
+        - variable: tcp
+          label: "TCP Ingress Configuration"
+          group: "Ingress"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enable TCP Ingress"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: type
+                      label: "Select Ingress Type"
+                      schema:
+                        type: string
+                        default: "TCP"
```

Comment:
I don't think 8080 is tcp, are you 100% sure?

Reply:
[UniFi - Ports Used](https://help.ui.com/hc/en-us/articles/218506997-UniFi-Ports-Used) states 8080 is TCP.

Reply:
Yeah, so is HTTP.

Reply:
Afaik it's still:
http://unifi:8080/inform

Reply:
[http://unifi:8080/inform](http://unifi:8080/inform) is the default resolution if there is no DHCP Option 43 configured.

Reply:
the */inform was not the relevant part, it's the fact the protocol seems to be http.

Reply:
I've looked some more into tcp vs http.
In our case, TCP might be the best one, because we forward all hostnames when using TCP ingress. So even IP traffic without hostname gets forwarded, wheras HTTP ingress is filtered based on hostname.